### PR TITLE
RELATED: CL-11683 Added support for identifier to uri conversion

### PR DESCRIPTION
**Pull request by vojty**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/183/ : SUCCESS in 31s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/184/ : SUCCESS in 45s

File: src/metadata.js [hunk 1]
```diff
@@ -546,6 +546,25 @@ export function getObjectUri(projectId, identifier) {
...
 /**
+ * Get uris specified by identifiers
+ *
+ * @method getIndetifiersForUris
+ * @param {String} projectId id of the project
+ * @param {Array} identifiers identifiers of the metadata objects
+ * @return {Array} array of identifier + uri pairs
+ */
+export function getIndetifiersForUris(projectId, identifiers) {
+    return ajax(`/gdc/md/${projectId}/identifiers`, {
```
Comment (vaclavbohac) [inline src/metadata.js]: Please use `post` instead of plain `ajax`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/185/ : SUCCESS in 2m 17s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/186/ : SUCCESS in 31s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/187/ : SUCCESS in 32s

File: src/metadata.js [hunk 2]
```diff
@@ -546,6 +546,24 @@ export function getObjectUri(projectId, identifier) {
...
 
 /**
+ * Get uris specified by identifiers
+ *
+ * @method getIndetifiersForUris
+ * @param {String} projectId id of the project
+ * @param {Array} identifiers identifiers of the metadata objects
+ * @return {Array} array of identifier + uri pairs
+ */
+export function getIndetifiersForUris(projectId, identifiers) {
```
Comment (vaclavbohac) [inline src/metadata.js]: `getUrisFromIdentifiers`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/188/ : SUCCESS in 31s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-js-merge-compliance-check/20/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/189/ : SUCCESS in 1m 23s

**Pull request merged**